### ISSUE TITLE
Cart pendulum ODE docstring?

### ISSUE THREAD

**Issue by Jacob-Stevens-Haas** [dynamicslab/pysindy#242]
The odes module doesn't have any docstrings.  In particular, I was trying to use [`pendulum_on_cart`](https://github.com/dynamicslab/pysindy/blob/32c96aa713288d74d744f97838901ff08c427695/pysindy/utils/odes.py#L396), but I'm not sure what the coordinates of `x` are supposed to represent.  I would guess that `x` is `[angular posit of pendulum, linear posit of cart, angular velocity of pendulum, linear velocity of cart]`, `M` is the mass of the cart,  `m` the mass of the pendulum weight,  `L` is length, and `F` is friction coefficient (`g=9.81` is obv, ofc).  I got different equations when I tried to calculate the equations of motion, so I'm not sure if I made a mistake in math or if I misinterpreted the variables.

@akaptano, looks like you added these lines, but the ODE isn't used in any of the example notebooks.  Do you know what it's supposed to mean?

**Comment by akaptano** [comment]
Yeah, this is a good idea in general to add some docstrings. That ODE in particular was added because it is what Kardy uses to showcase SINDy-PI. Unfortunately there are some typos in Kardy's derivation I think (and maybe also in mine), so you need to make sure the equations are right. If you post your derived equations here we can compare.

**Comment by akaptano** [comment]
On this note, I have an undergraduate working with the dysts database (https://github.com/williamgilpin/dysts) which would be a really nice database to integrate better with PySINDy for users that want to mess around with system identification and hyper parameter scanning. Would be good to brainstorm best way to do this.
